schemas/2015-05-21-preview/Microsoft.DevTestLabs.json

```diff
@@ -0,0 +1,69 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-05-21-preview/Microsoft.DevTestLab.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.DevTestLabs",
+  "description": "Azure DevTest Labs Resource Types",
+  "definitions": {
+    "labs": {
+      "description": "Required properties to create a DevTest Lab",
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "Microsoft.DevTestLabs/labs"
```

Comment:
ARM has a RP whose namespace is Microsoft.DevTestLab, please fix this everywhere.